src/Nethermind/nuget.config

```diff
@@ -3,14 +3,14 @@
   <packageSources>
     <clear />
     <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
-    <!--<add key="nugettest.org" value="https://apiint.nugettest.org/v3/index.json" />-->
+    <add key="nugettest.org" value="https://apiint.nugettest.org/v3/index.json" />
```

Comment:
This should not be merged. I'll prepare what's needed.

Reply:
@kamilchodola [Here](https://github.com/NethermindEth/nethermind/pull/5976) you go